starboard/shared/starboard/player/filter/testing/video_decoder_test.cc

```diff
@@ -160,7 +160,7 @@ TEST_P(VideoDecoderTest, ThreeMoreDecoders) {
               kSbMediaAudioCodecNone};
           PlayerComponents::Factory::CreationParameters creation_parameters(
               CreateVideoStreamInfo(fixture_.dmp_reader().video_codec()),
-              &players[i], output_mode,
+              &players[i], output_mode, /* max_input_size */ -1,
```

Comment:
Ideally, we should also have tests with non default |max_input_size|.

Reply:
Added test cases.